Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -190,12 +192,16 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 			base.OnElementPropertyChanged(sender, e);
 
 			if (e.PropertyName == nameof(TabbedPage.CurrentPage))
+			{
 				if(Element.CurrentPage != null)
```

Comment:
If you happen to need to make another change to this PR, want to add the missing space here, too? :)